fiftyone/utils/sam.py

```
@@ -47,6 +51,516 @@ def __init__(self, d):
+        box_transform (None): SAM specific box transform function to apply
+        point_transform (None): SAM specific point transform function to apply
+    """
+
+    def __init__(
+        self,
+        field_mapping=None,
+        transform=None,
+        use_numpy=None,
+        box_transform=None,
+        point_transform=None,
+        **kwargs,
```

Ruff (0.15.6)

[warning] Missing type annotation for `**kwargs`

(ANN003)